test/EndToEnd/tests/PackageNameSpaceTests.ps1

```diff
@@ -326,6 +326,296 @@ function Test-VsPackageInstallerServices-PackageNamespaceInstall-WithMultipleFee
     }
 }
 
+function Test-PackageNamespaceInstall_Succeed
```

Comment:
Ok. I added project type "PC" into all of them. I chosen `PC` over `PackagesConfig` otherwise some test name would be very long.
It looks I didn't add 'PMC install' tests into End2end test previously, but I did add Apex tests with  https://github.com/NuGet/NuGet.Client/pull/4150